### PR TITLE
Change implementation of file_rec source to accept positional parameter

### PR DESCRIPTION
**Pull request by ohcibi**

### CODE REVIEW

File: doc/denite.txt [hunk 1]
```diff
@@ -148,7 +148,7 @@ EXAMPLES						*denite-examples*
 	" Define alias
 	call denite#custom#alias('source', 'file_rec/git', 'file_rec')
 	call denite#custom#var('file_rec/git', 'command',
-	      \ ['git', 'ls-files', '-co', '--exclude-standard'])
+	      \ ['git', 'ls-files', '-co', '--exclude-standard', ':directory'])
```
Comment (Shougo) [inline doc/denite.txt]: I think find command customization must be described in example section.

**Comment by ohcibi** [comment]
I've updated the commit

**Comment by Shougo** [comment]
No difference?
I have not seen the change.

**Comment by ohcibi** [comment]
hm.. me neither 8-) i forgot to `git add` before `commit --amend` 8-). should be alright now

File: doc/denite.txt [hunk 2]
```diff
@@ -89,11 +89,11 @@ EXAMPLES						*denite-examples*
 >
 	" Change file_rec command.
 	call denite#custom#var('file_rec', 'command',
-	\ ['ag', '--follow', '--nocolor', '--nogroup', '-g', ''])
+	\ ['ag', '--follow', '--nocolor', '--nogroup', '-g', '', ':directory'])
```
Comment (Shougo) [inline doc/denite.txt]: Please remove `':directory'`.
It is no needed.

File: doc/denite.txt [hunk 3]
```diff
@@ -89,11 +89,11 @@ EXAMPLES						*denite-examples*
...
 	" For ripgrep
 	" Note: It is slower than ag
 	call denite#custom#var('file_rec', 'command',
-	\ ['rg', '--files', '--glob', '!.git'])
+	\ ['rg', '--files', '--glob', '!.git', ':directory'])
```
Comment (Shougo) [inline doc/denite.txt]: Please remove `':directory'`.
It is not needed.

File: doc/denite.txt (hunk 1 above)
Comment (Shougo) [inline doc/denite.txt]: Above same.

**Comment by Shougo** [comment]
Please fix the examples.

**Comment by ohcibi** [comment]
Done

**Comment by Shougo** [comment]
LGTM

**Pull request merged**